services/src/main/java/io/scalecube/services/ServiceRegistry.java

```diff
@@ -29,4 +30,6 @@
   
   Cluster cluster();
 
+  Transport transport();
```

Comment:
Also I don't think it is totally good idea to expose from this interface Transport, btw the same is true about Cluster also. It seems a little bit awkward that ServiceRegistry interface exposes Cluster/Transport.